hswindow/hseditor/hsdt1editor/dt1_editor.go

```diff
@@ -113,26 +112,29 @@ func (e *DT1Editor) UpdateMainMenuLayout(l *g.Layout) {
 	*l = append(*l, m)
 }
 
-// RegisterKeyboardShortcuts register a new keyboard shortcut
-// nolint:wsl // I can't put this ccommented out code anywhere else
-func (e *DT1Editor) RegisterKeyboardShortcuts(inputManager *hsinput.InputManager) {
-	// Ctrl+Shift+S saves file
-	inputManager.RegisterShortcut(func() {
-		e.Save()
-	}, g.KeyS, g.ModShift+g.ModControl, false)
-
-	// nolint:gocritic // we may want to use this code
-	/*
-		// right arrow goes to the next tile group
-		inputManager.RegisterShortcut(func() {
-			e.dt1Viewer.SetTileGroup(e.dt1Viewer.TileGroup() + 1)
-		}, g.KeyRight, g.ModNone, false)
-
-		// left arrow goes to the previous tile group
-		inputManager.RegisterShortcut(func() {
-			e.dt1Viewer.SetTileGroup(e.dt1Viewer.TileGroup() - 1)
-		}, g.KeyLeft, g.ModNone, false)
-	*/
+// KeyboardShortcuts register a new keyboard shortcut
+func (e *DT1Editor) KeyboardShortcuts() []g.WindowShortcut {
+	return []g.WindowShortcut{
+		/*
+			// right arrow goes to the next tile group
+			giu.WindowShortcut{
+				Callback: func() {
+					e.dt1Viewer.SetTileGroup(e.dt1Viewer.TileGroup() + 1)
+				},
+				Key:      g.KeyRight,
+				Modifier: g.ModNone,
+			},
+
+			// left arrow goes to the previous tile group
+			giu.WindowShortcut{
+				Callback: func() {
+					e.dt1Viewer.SetTileGroup(e.dt1Viewer.TileGroup() - 1)
+				},
+				Key:      g.KeyLeft,
+				Modifier: g.ModNone,
+			},
+		*/
```

Comment:
should we delete this?

Reply:
not really sure. I opened an issue: https://github.com/OpenDiablo2/HellSpawner/issues/329

Reply:
not really sure... we may want to use this code. I'll open an issue for it

Reply:
okay, as long as we have an issue for it this is fine.